server/accounts.go

```diff
@@ -4114,18 +4120,65 @@ type transform struct {
 	src, dest string
 	dtoks     []string
 	stoks     []string
-	dtpi      []int8
+	dtpi      [][]int // destination token position indexes
+	dtpinp    []int32 // destination token position index number of partitions
```

Comment:
we need these comments on all these 100s of short named variables, nice one :)

Reply:
Thanks. Yes all those short named variables don't make it easy to get up to speed on what the code does, I kept renaming 'dpi' to a long variable name as I was figuring out one by one what each letter in the short variable name stood for, and having those long variable names made it much easier to read and understand the source code. Then at the end I renamed them back to short names but did add the comments as it will greatly help (IMHO) others that may want to read the code for those functions and understand it.

One could probably spend quite a bit of time going over all of the server code and adding comments like these :)